src/components/drawing/index.js

```diff
@@ -579,16 +579,17 @@ drawing.makeTester = function(gd) {
         tester.node()._cache = {};
     }
 
-    gd._tester = tester;
-    gd._testref = testref;
+    drawing.tester = gd._tester = tester;
+    drawing.testref = gd._testref = testref;
```

Comment:
Sure. child window behavior is strange... like it looks like in IE and Edge we have some even bigger problems with it.

FWIW I think the `layout.title` issue you flagged above is not a bounding box issue, but the fact that the two `<svg>` layers are somehow not on top of each other, the front layer appears down below. the main one.